book/src/offline-signing/durable-nonce.md

```diff
@@ -0,0 +1,201 @@
+# Durable Transaction Nonces
+
+Durable transaction nonces are a mechanism for getting around the typical
+short lifetime of a transaction's [`recent_blockhash`](../transaction.md#recent-blockhash).
+They are implemented as a Solana Program, the mechanics of which can be read
+about in the [proposal](../implemented-proposals/durable-tx-nonces.md).
+
+## Known Issues
+
+### Fee Theft Opportunity
+
+The durable nonce implementation contains a vulernability which allows for fees
+to be stolen by a transaction using the feature under certain conditions. To
+achieve this, the transaction must have failed with an instruction error, which
+rolls back the step that advanced the stored nonce, allowing the transaction to
+be replayed and fees charged. This can be repeated until the stored nonce is
+successfully advanced.
+
+#### - Mitigation
+
+To minimize loss of funds, a low-balance account should always be used to pay
```

Comment:
Passive voice nit:
"To minimize loss of funds, use a low-balance account to pay..."